docs/main.html

```diff
@@ -2,4 +2,78 @@
 lhn_content: 'homepage'
 ---
 
-<!-- Add Homepage content here -->
+<div class="homepage">
+
+    <h1 class="title">
+        Welcome to Expensify Help!
+    </h1>
+
+    <p>
+        Find all the answers to all your Expensify questions, as well as support for user and admin questions from the
+        Expensify Community.
+    </p>
+
+    <h2>
+        Which best describes how you use Expensify?
+    </h2>
+
+    <div class="cards-group">
+
+        <div class="card send-money" onclick="navigateTo('/hubs/send-money')">
+            <div class="left-icon">
+                <img id="icon" src="/assets/images/shield.svg" />
+            </div>
+            <div class="body">
+                <div class="title">Send money</div>
+                <div class="description">Brief description of who would fall under this category.</div>
+            </div>
+        </div>
+
+        <div class="card request-money" onclick="navigateTo('/hubs/request-money')">
+            <div class="left-icon">
+                <img id="icon" src="/assets/images/users.svg" />
+            </div>
+            <div class="body">
+                <div class="title">Request money</div>
+                <div class="description">Brief description of who would fall under this category.</div>
+            </div>
+        </div>
+
+    </div>
+
+    <h2>
+        Our most popular guides and tutorials
+    </h2>
+
+    <div class="cards-group">
+
+        <div class="card" onclick="navigateTo('/main')">
+            <div class="body">
+                <div class="title">Getting started</div>
+                <div class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur hendrerit euismod dui.</div>
+            </div>
+        </div>
+
+        <div class="card" onclick="navigateTo('/main')">
+            <div class="body">
+                <div class="title">Getting started</div>
+                <div class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur hendrerit euismod dui.</div>
+            </div>
+        </div>
+
+        <div class="card" onclick="navigateTo('/main')">
+            <div class="body">
+                <div class="title">Getting started</div>
+                <div class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur hendrerit euismod dui.</div>
+            </div>
+        </div>
+
+        <div class="card" onclick="navigateTo('/main')">
```

Comment:
I don't like having the JS and the HTML this tightly coupled. It could be a real mess someday. What I would prefer to do is add a class like `.navigationButton` and use a data attribute like `data-href="/main"`).

Then in JS, attach a click handler to all `.navigationButton` and read the destination from the `data-href` attribute.

Could we please do that instead?

An alternative to that may be... instead of these being clickable divs, make them anchors with `display: block`.

Reply:
Oh that's right, I went for the anchors route which is a cleaner solution, thanks!